Formula/azure-cli.rb

```diff
@@ -784,3 +791,18 @@ def install
     assert_equal azure_cloud["endpoints"]["resourceManager"], "https://management.azure.com/"
   end
 end
+
+__END__
+diff --git a/tools/setup.py b/tools/setup.py
```

Comment:
`tools/setup.py` is actually not used as part of Azure CLI installation at all, so this patch is unnecessary.